server/src/controllers/Events/resolver.ts

```diff
@@ -324,6 +324,19 @@ export class EventResolver {
       where: { user_id_event_id: { user_id: userId, event_id: eventId } },
       include: { event: true },
```

Comment:
```suggestion
      include: { event: true, user: true },
```
Gets also user from the relation, event was already included.